Update README (#1)
DevExpress-Examples/aspxgridview-delete-confirm-dialog-with-dont-ask-me-again-option: Readme.md

**[[Run Online]](https://codecentral.devexpress.com/128540794/)**
<!-- run online end -->

> This example demonstrates how to implement a delete confirmation dialog with an option to prevent its displays for subsequent grid row deletions.

Reviewer: ```suggestion
This example demonstrates how to implement a delete confirmation dialog with an option to prevent its display for subsequent grid row deletions.
```

![Delete Confirmation Dialog](./delete-confirm-dialog.png)

## Implementation Details

The confirm dialog is based on the [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl). The popup contains the dialog content wrapped in a WebUserControl.

> > **Note:** In this example, the "Don't ask confirmation" check box's value is stored in a client JavaScript variable (`dontAskConfirmation`). We recommend that in a real-life application you save this value to a user-specific storage such as cookies or user profile settings.

Reviewer: 1. `**Note:** In this example, the "Don't ask confirmation" check box's value is stored in a client JavaScript variable (`dontAskConfirmation`).`

Is the "Don't ask confirmation" string editable? If so, can you change it to "Don't ask for confirmation"?

2. `We recommend that in a real-life application you save this value to a user-specific storage such as cookies or user profile settings.`

Please add a comma after the word "application".

Author: Done

Reviewer: Thanks

The example application uses the following techniques:

- The `ID` field value of the processed row is stored in the client `rowVisibleIndex` variable.

- The ASPxPopupControl is made transparent so that it displays its child [ASPxRoundPanel](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxRoundPanel) as the dialog's immediate container.

- The ASPxPopupControl is used in modal mode and configured to always be displayed at the Grid View's center.

> - The "Yes" button is made the dialog's default button (its ID is assigned to the ASPxRoundPanel's [DefaultButton](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPanelBase.DefaultButton) property) so that you can press  `Enter` to submit the dialog. The "Yes" button programmatically receives the input focus after the dialog is shown.

Reviewer: I'd like to update the first part of the first sentence ("is made"). Also, do users press the Enter key? If so, do you mean:

```suggestion
- The "Yes" button is the dialog's default button (its ID is assigned to the ASPxRoundPanel's [DefaultButton](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPanelBase.DefaultButton) property), so users can press `Enter` to submit the dialog. The "Yes" button programmatically receives the input focus after the dialog is shown.
```
